src/tools/other/point2grid/point2grid.cc

```diff
@@ -1494,19 +1495,10 @@ static void process_point_nccf_file(NcFile *nc_in, MetConfig &config,
          }
       }
 
```

Comment:
Need to reset `var_cell_mapping` which overrides `cell_mapping` because of different lat/lon variables for the specific data variables.
- find the lat/lon variables and build `cell_mapping`.
- check the `coordinates` attribute of the data variable and compare the lat/lon variables with the default lat/lon
- build `var_cell_mapping` and apply `var_cell_mapping` instead of the default `cell_mapping` if the lat/lon variables from the `coordinates` attribute are different.

`var_cell_mapping` from the previous variable should not be applied for the next variable

Reply:
Thanks @hsoh-u, I pushed a commit to clear the var_cell_mapping vector at the top of each loop iteration. As we discussed on Slack, that should do it.